styles/navigation.css

```diff
@@ -0,0 +1,110 @@
+.topnav__container--std {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  background-color: var(--primary-color);
+  transition: top 0.2s ease-in-out;
+}
+
+.topnav__container--std .nav--std {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  padding: 1rem;
+}
+
+.topnav__container--std ul {
+  list-style: none;
+  display: inline-flex;
+  flex: 1 1 auto;
+  align-items: center;
+}
+
+.topnav__list--align-start {
+  justify-content: flex-start;
+}
+
+.topnav__list--align-end {
+  justify-content: flex-end;
+}
+
+.topnav__title a {
+  color: var(--base-color);
+}
+
+.topnav__btn {
+  position: relative;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
```

Comment:
you can remove whitespaces if you forgot

Reply:
When I have a lot of styles in a component I prefer to separate display, position and other properties using a whitespace line between them. But I can change it if you say so.